src/Relay/Connection/Paginator.php

```diff
@@ -83,7 +85,7 @@ public function forward(ArgumentInterface $args)
     /**
      * @param int|callable $total
      *
-     * @return ConnectionInterface|object A connection or a promise
```

Comment:
see previous

Reply:
Tested this method with Xdebug and fixed to Connection|Promise